apps/host/src/components/modules/challenges/challenge-grid/challenge-grid.module.scss

```diff
@@ -1,6 +1,6 @@
 .filterOptionWrapper {
   display: grid;
-  grid-template-columns: repeat(auto-fill, minmax(300px, 1fr));
+  grid-template-columns: repeat(auto-fill, minmax(290px, 1fr));
```

Comment:
any reason for this change

Reply:
New challenge was checkbox is not adjusting in the same row and filters are going down second row